lightning/config/global.go

```diff
@@ -144,7 +144,7 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 
 	logLevel := flagext.ChoiceVar(fs, "L", "", `log level: info, debug, warn, error, fatal (default info)`, "", "info", "debug", "warn", "warning", "error", "fatal")
 	logFilePath := fs.String("log-file", "", "log file path")
-	redactLog := fs.Bool("redact-log", false, "whether to redact sensitive info in log")
+	redactLog := fs.Bool("redact-info-log", false, "whether to redact sensitive info in log")
```

Comment:
We need to preserve compatibility for 5.0-rc

Reply:
shall we also cherry-pick this branch to release-5.0-rc?

Reply:
We should pick both this commit and https://github.com/pingcap/tidb-lightning/pull/538 to 5.0-rc. But we can do it in next version.

Reply:
oh, we only need to cherry-pick this commit to release-4.0